src/utstring.h

```diff
@@ -121,6 +121,15 @@ do {                                                             \
 
 #define utstring_body(s) ((s)->d)
 
+static char* utstring_release(UT_string *s) {
```

Comment:
I have done this kind of thing before manually. So, I think it's a useful operation. Agreed on the need for `_UNUSED_`. I think the name `_release` is so-so. I want to imply that it's releasing it "to" the caller. Like `utstring_disown`. Thoughts? Incidentally I once or twice wrote the opposite function that adopts the string you give it and makes a utstring around it. From that point of view, `utstring_own` and `utstring_disown` are a pair.